### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by HaydenBS93**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+          >
+          <a
+            href="#"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Blog</a
+          >
+          <a
+            href="#"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Contact</a
```
Comment (naha92) [inline index.html:54]: Missing links to pages

File: index.html [hunk 2]
```diff
@@ -0,0 +1,521 @@
...
+          </a>
+        </div>
+      </div>
+    </header>
+    <main>
+      <!-- Product Grid -->
+      <section class="max-w-7xl mx-auto px-4 py-12">
+        <div class="grid grid-cols-3 gap-10 py-6">
+          <div class="group relative cursor-pointer">
+            <a href="#">
```
Comment (naha92) [inline index.html:85]: Link to shop

File: index.html [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+                />
+                <span
+                  class="absolute bottom-0 left-0 bg-black text-xl text-white px-4 py-2 rounded-bl-lg"
+                  >Travel Tech</span
+                >
+              </div>
+            </a>
+          </div>
+          <div class="group relative cursor-pointer">
+            <a href="#">
```
Comment (naha92) [inline index.html:101]: Link to shop missing

File: index.html [hunk 4]
```diff
@@ -0,0 +1,521 @@
...
+            </button>
+          </a>
+        </div>
+      </section>
+      <!-- Product Feature section -->
+      <section
+        id="product-of-the-week"
+        class="py-12 bg-gradient-to-b from-gray-100 to-white"
+      >
+        <a href="#">
```
Comment (naha92) [inline index.html:149]: Link to specific product/shop missing

File: index.html [hunk 5]
```diff
@@ -0,0 +1,521 @@
...
+            <div class="p-4 space-y-4">
+              <span>Full-time designer</span>
+              <p class="text-sm">
+                Quos sunt ad dolore ullam qui. Enim et quisquam dicta molestias.
+                Corrupti quo voluptatum eligendi autem labore.
+              </p>
+              <div class="flex space-x-8">
+                <span>£80,000</span>
+                <span>Manchester, UK</span>
+              </div>
```
Comment (naha92) [inline index.html:233]: Unsure what we agreed on - should individual blog posts be clickable here?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,521 @@
...
+                <span>Judith Black</span>
+                <span>CEO of Tuple</span>
+              </div>
+            </figure>
+          </div>
+        </div>
+      </section>
+      <!-- CTA Banner section-->
+      <section class="max-w-screen-2xl mx-auto mt-32 relative">
+        <a href="#">
```
Comment (naha92) [inline index.html:293]: Intentional design choice to have the entire CTA clickable, or should we just have the button clickable?
Missing link to shop page

File: index.html [hunk 7]
```diff
@@ -0,0 +1,521 @@
...
+            src="/assets/images/tiffany-nutt-0ClfreiNppM-unsplash 1.png"
+            alt="Orange Bicycle"
+            class="inset-0 w-full h-full object-cover"
+          />
+          <div class="cta-textbox absolute">
+            <h2 class="text-5xl text-travel-orange font-bold pb-8 max-w-lg">
+              Our biggest sale is now live!
+            </h2>
+            <div class="mt-16">
+              <a href="#">
```
Comment (naha92) [inline index.html:304]: Missing link to shop page

File: index.html [hunk 8]
```diff
@@ -0,0 +1,521 @@
...
+                required
+                class="flex-auto rounded-md px-3 py-2 text-black shadow-sm ring-1 ring-inset ring-travel-orange focus:ring-2 focus:ring-inset focus:ring-travel-orange md:text-sm border-travel-orange"
+                placeholder="Enter your email"
+              />
+              <button
+                type="submit"
+                class="flex-none rounded-md bg-travel-orange px-3.5 py-2.5 text-white text-sm font-semibold shadow-sm hover:bg-orange-600 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-yellow-500"
+              >
+                Subscribe
+              </button>
```
Comment (naha92) [inline index.html:341]: In future probably want this to redirect to a success page - potentially add a 'Registration Successful' popup

File: index.html [hunk 9]
```diff
@@ -0,0 +1,521 @@
...
+              height="48"
+            />
+            <img
+              id="statamic-logo"
+              class="col-span-2 col-start-2 max-h-12 w-full object-contain md:col-start-auto lg:col-span-1"
+              src="https://tailwindui.com/img/logos/158x48/statamic-logo-gray-900.svg"
+              alt="Statamic"
+              width="158"
+              height="48"
+            />
```
Comment (naha92) [inline index.html:403]: Links to companies missing

File: index.html [hunk 10]
```diff
@@ -0,0 +1,521 @@
...
+        class="fixed inset-x-0 z-10 bottom-0 flex flex-col justify-between gap-x-8 gap-y-4 p-4 bg-white border-t border-t-travel-orange md:flex-row md:items-center lg:px-8 opacity-0 transition-opacity duration-500 ease-in-out"
+      >
+        <p class="max-w-4xl text-sm text-black">
+          Discover our curated selection of travel essentials, gear, and
+          accessories—an ensemble designed to enhance your journeys and enrich
+          your experiences on the road. Embrace these offerings at your
+          discretion, knowing they are crafted to elevate your adventures with
+          both practicality and pleasure. See our
+          <a
+            href="/privacy-policy.html"
```
Comment (naha92) [inline index.html:428]: Wrong link: cookie-policy.html

File: index.html [hunk 11]
```diff
@@ -0,0 +1,521 @@
...
+        <div class="pb-6">
+          <a
+            href="#"
+            class="text-lg font-semibold text-travel-orange hover:text-orange-600"
+            >Blog</a
+          >
+        </div>
+        <div class="pb-6">
+          <a
+            href="#"
```
Comment (naha92) [inline index.html:483]: Missing links to pages

File: index.html [hunk 12]
```diff
@@ -0,0 +1,521 @@
...
+        </a>
+        <a href="#" class="text-gray-400 hover:text-gray-500">
+          <span class="sr-only">Pinterest</span>
+          <img src="/assets/icons/pinterest.svg" alt="Pinterest icon" />
+        </a>
+        <a href="#" class="text-gray-400 hover:text-gray-500">
+          <span class="sr-only">YouTube</span>
+          <img src="/assets/icons/youtube.svg" alt="Youtube icon" />
+        </a>
+        <a href="#" class="text-gray-400 hover:text-gray-500">
```
Comment (naha92) [inline index.html:510]: Missing links to socials

File: index.html [hunk 13]
```diff
@@ -0,0 +1,521 @@
...
+          <span class="sr-only">Instagram</span>
+          <img src="/assets/icons/instagram.svg" alt="Instagram icon" />
+        </a>
+      </div>
+      <p class="mt-10 text-center text-lg text-black">
+        &copy; 2020 TravelUp, Inc. All rights reserved.
```
Comment (naha92) [inline index.html:516]: 2024

File: index.html (hunk 1 above)
Comment (HaydenBS93) [reply index.html:54]: done

File: index.html [hunk 14]
```diff
@@ -0,0 +1,521 @@
...
+          alt="Hero image"
+          class="w-full h-96 object-cover"
+        />
+        <p
+          class="max-w-xl mx-auto absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 text-3xl text-white text-center px-4 py-2 rounded-lg z-10"
+        >
+          Shop our travel item trends throughout all of our product range
+        </p>
+        <div class="absolute bottom-16 w-full text-center">
+          <a href="#">
```
[the remark this answers is not in the document]
Comment (HaydenBS93) [reply index.html:70]: done

File: index.html (hunk 2 above)
Comment (HaydenBS93) [reply index.html:85]: done

File: index.html (hunk 3 above)
Comment (HaydenBS93) [reply index.html:101]: done

File: index.html [hunk 15]
```diff
@@ -0,0 +1,521 @@
...
+                />
+                <span
+                  class="absolute bottom-0 left-0 bg-black text-xl text-white px-4 py-2 rounded-bl-lg"
+                  >Accessories</span
+                >
+              </div>
+            </a>
+          </div>
+          <div class="group relative cursor-pointer">
+            <a href="#">
```
[the remark this answers is not in the document]
Comment (HaydenBS93) [reply index.html:117]: done

File: index.html (hunk 4 above)
Comment (HaydenBS93) [reply index.html:149]: done

File: index.html [hunk 16]
```diff
@@ -0,0 +1,521 @@
...
+                Corrupti quo voluptatum eligendi autem labore.
+              </p>
+              <div class="flex space-x-8">
+                <span>£80,000</span>
+                <span>Manchester, UK</span>
+              </div>
+            </div>
+            <div class="flex justify-end items-center mt-4 space-x-2">
+              <button class="text-travel-orange text-base font-semibold">
+                View more posts
```
[the remark this answers is not in the document]
Comment (HaydenBS93) [reply index.html:237]: done

File: index.html (hunk 5 above)
Comment (HaydenBS93) [reply index.html:233]: amended these to suit more blog posts

File: index.html (hunk 8 above)
Comment (HaydenBS93) [reply index.html:341]: Yeah we can add this as tech debt in a new branch if you want to action this?

File: index.html (hunk 10 above)
Comment (HaydenBS93) [reply index.html:428]: Spoke to you about this, confirmed we will keep the same name as it will under the same part

File: index.html (hunk 9 above)
Comment (HaydenBS93) [reply index.html:403]: Agreed to keep as images

File: index.html (hunk 12 above)
Comment (HaydenBS93) [reply index.html:510]: done

File: index.html (hunk 11 above)
Comment (HaydenBS93) [reply index.html:483]: done

File: index.html (hunk 13 above)
Comment (HaydenBS93) [reply index.html:516]: done

File: index.html (hunk 7 above)
Comment (HaydenBS93) [reply index.html:304]: done

File: index.html (hunk 6 above)
Comment (HaydenBS93) [reply index.html:293]: Intentional choice to have whole as clickable, can complete

**Review by naha92 — APPROVED** [review]

**Review by naha92 — APPROVED** [review]

**Pull request merged**